### ISSUE TITLE
File explorer Window is wrapped when clicking Browse in import dailog, missing icons and unable to import projects.

### ISSUE THREAD

**Issue by Anutrix** [godotengine/godot#37401]
**Godot version:**
master build https://github.com/godotengine/godot/commit/fafb0ba06df9417a22bd8deb31a61feeb24de794

**OS/device including version:**
Window 10

**Issue description:**

- [x] Issue 1: File explorer Window is wrapped when clicking Browse in import dialog.
- [x] Issue 2: Resizing to see file list also reveals that icons for files and folders are missing.
- [x] Issue 3: Import .project files does not open project.

I made a single issue since they seem related.

```
ERROR: Condition "!windows.has(p_window)" is true.
   at: window_resize (drivers\vulkan\vulkan_context.cpp:722)
```
is also seen in the console every time I interact with a window such as opening a folder or changing drive.

**Steps to reproduce:**
1. Open Godot.
2. Click Import.
3. Click Browse. - Issue 1 is observed.
4. Resize to seem files. - Issue 2 is observed.
5. Select any .project file to import. - Issue 3 is observed.

**Comment by Anutrix** [comment]
When popups weren't Windows(pre-https://github.com/godotengine/godot/commit/441f1a5fe9a3bf0e4e5dab578f793500b1ff6e3d), they used to be x%(I think 80)/similar of the height and width of the parent window. In other words, they were dependent on parent windows.
But now since they are independent, they use min-height and min-width leading to things like `Tree`s which don't have min size by default to shrunk to size 0. This is the reason for Issue 1.

**Comment by CNate** [comment]
The Vulkan layers can be fixed by installing the SDK. I am also encountering the resizing error on top of a few other things
![image](https://user-images.githubusercontent.com/13683291/78721906-67623a00-78f6-11ea-845e-128bb8471dc5.png)

**Comment by Calinou** [comment]
For the record, this also occurs with the Quick Open dialogs:

![Quick Open Scene dialog](https://user-images.githubusercontent.com/180032/79700537-aac58c80-8296-11ea-9999-bf2bfc9a3070.png)

**Comment by akien-mga** [comment]
Issue 3 should be fixed by #39993 which supersedes #39145, so all issues are fixed.
